File: php/Film.php
<?php

// Class Film merepresentasikan entity bioskop
class Film {
    private $id;
    private $judul;
    private $genre;
    private $harga;
    private $poster;

    public function __construct($id = "", $judul = "", $genre = "", $harga = 0, $poster = "") {
        $this->id = $id;
        $this->judul = $judul;
        $this->genre = $genre;
        $this->harga = $harga;
        $this->poster = $poster;
    }

    // Getter
    public function getId() { return $this->id; }
    public function getJudul() { return $this->judul; }
    public function getGenre() { return $this->genre; }
    public function getHarga() { return $this->harga; }
    public function getPoster() { return $this->poster; }

    // Setter
    public function setId($id) { $this->id = $id; }
    public function setJudul($judul) { $this->judul = $judul; }
    public function setGenre($genre) { $this->genre = $genre; }
    public function setHarga($harga) { $this->harga = $harga; }
    public function setPoster($poster) { $this->poster = $poster; }
}

File: java/Film.java
// Class Film untuk merepresentasikan data film di bioskop
public class Film {
    // Atribut privat (Encapsulation)
    private String id;
    private String judul;
    private String genre;
    private int harga;

    // Constructor Kosong
    public Film() {

    }

    // Constructor Berparameter untuk inisialisasi atribut
    public Film(String id, String judul, String genre, int harga) {
        this.id = id;
        this.judul = judul;
        this.genre = genre;
        this.harga = harga;
    }

    // Getter
    public String getId() { 
        return id; 
    }
    public String getJudul() { 
        return judul; 
    }
    public String getGenre() { 
        return genre; 
    }
    public int getHarga() { 
        return harga; 
    }

    // Setter
    public void setId(String id) { 
        this.id = id; 
    }
    public void setJudul(String judul) { 
        this.judul = judul; 
    }
    public void setGenre(String genre) { 
        this.genre = genre; 
    }
    public void setHarga(int harga) { 
        this.harga = harga; 
    }
}